.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,15 +1,25 @@
-<!--
-Thanks for reporting a problem with Airsonic! Please complete all of the
-following sections to make it easier to assist you.
--->
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'in: UNKNOWN-frontend, status: waiting-for-feedback, type: bug'
```

Comment:
Are bug reports always for frontend?

Reply:
We can also make it the default value by creating a label such as "Please select" or "UNKNOWN". as we like...